Main/Source/miscitem.cpp

```diff
@@ -4015,3 +4019,20 @@ alpha mangoseedling::GetOutlineAlpha(int Frame) const
   Frame &= 31;
   return 50 + (Frame * (31 - Frame) >> 1);
 }
+
+alpha skeletonkey::GetOutlineAlpha(int Frame) const
+{
+  Frame &= 31;
+  return Frame * (31 - Frame) >> 1;
+}
+
+col16 skeletonkey::GetOutlineColor(int Frame) const
+{
```

Comment:
if(!IsBroken())

Reply:
Although keys don't break, this is a useful way to create a default return value so that clang C++ compilation works.